ko/news/_posts/2019-11-23-ruby-2-7-0-preview3-released.md

```diff
@@ -280,52 +260,43 @@ The final decision is not made, but will be fixed by the official release.
     * open3
     * pstore
     * singleton
-  * The following default gems were only promoted at ruby-core,
-    but not yet published on rubygems.org.
+  * 다음 기본 젬들은 ruby-core에서는 젬으로 승격되었지만, 아직 rubygems.org에서 배포하고 있지 않습니다.
     * monitor
     * observer
     * timeout
     * tracer
     * uri
     * yaml
 
-* `Proc.new` and `proc` with no block in a method called with a block
-   is warned now.
+* 블록을 넘긴 메서드의 호출 안에서 블록이 없는 `Proc.new`와 `proc`을 사용하면 경고가 발생합니다.
 
-* `lambda` with no block in a method called with a block errs.
+* 블록을 넘긴 메서드의 호출 안에서 블록이 없는 `lambda`는 에러가 발생합니다.
 
-* Update Unicode version and Emoji version from 11.0.0 to 12.0.0.
+* 유니코드와 에모지의 버전을 11.0.0에서 12.0.0으로 업데이트했습니다.
   [[Feature #15321]](https://bugs.ruby-lang.org/issues/15321)
 
-* Update Unicode version to 12.1.0,
-  adding support for U+32FF SQUARE ERA NAME REIWA.
+* 유니코드를 일본의 새로운 연호 레이와를 가리키는 코드(U+32FF SQUARE ERA NAME REIWA)에 대한 지원을 추가한 12.1.0으로 업데이트했습니다.
   [[Feature #15195]](https://bugs.ruby-lang.org/issues/15195)
 
-* `Date.jisx0301`, `Date#jisx0301`, and `Date.parse` support the
-  new Japanese era.
+* `Date.jisx0301`, `Date#jisx0301`, `Date.parse`에서 새 일본 연호를 지원합니다.
   [[Feature #15742]](https://bugs.ruby-lang.org/issues/15742)
 
-* Require compilers to support C99
+* 루비 빌드에 C99를 지원하는 컴파일러를 요구합니다.
   [[Misc #15347]](https://bugs.ruby-lang.org/issues/15347)
-  * Details of our dialect:
-    <https://bugs.ruby-lang.org/projects/ruby-master/wiki/C99>
+  * 이에 대한 자세한 정보: <https://bugs.ruby-lang.org/projects/ruby-master/wiki/C99>
 
-* ~~`Regexp#match{?}` with `nil` raises TypeError as String, Symbol.
-  [[Feature #13083]](https://bugs.ruby-lang.org/issues/13083)~~ reverted
+* ~~`Regexp#match{?}`에 `nil`을 넘기면 String, Symbol에서 처럼 TypeError를 던집니다.
+  [[Feature #13083]](https://bugs.ruby-lang.org/issues/13083)~~ 취소되었습니다.
 
-3895 files changed, 213426 insertions(+), 96934 deletions(-)
+[NEWS](https://github.com/ruby/ruby/blob/v2_7_0_preview3/NEWS)나
+[커밋 로그](https://github.com/ruby/ruby/compare/v2_6_0...v2_7_0_preview3)에서
+더 자세한 설명을 확인할 수 있습니다.
 
-See [NEWS](https://github.com/ruby/ruby/blob/v2_7_0_preview3/NEWS)
-or [commit logs](https://github.com/ruby/ruby/compare/v2_6_0...v2_7_0_preview3)
-for more details.
+이러한 변경사항에 따라, 루비 2.6.0 이후로 [파일 3895개 수정, 213426줄 추가(+), 96934줄 삭제(-)](https://github.com/ruby/ruby/compare/v2_6_0...v2_7_0_preview3)가 이루어졌습니다!
 
-With those changes,
-[stat](https://github.com/ruby/ruby/compare/v2_6_0...v2_7_0_preview3)
```

Comment:
This one is hard to translate with good wording, so I merged with L316 ;)
or plz suggest another way.